examples/misc/combined_table/combined_table_client.py

```diff
@@ -0,0 +1,48 @@
+
+""" A package that provides a CombinedTable class for pydeephaven clients
+that combines a historical and live table into a single table.
+
+See: https://deephaven.io/enterprise/docs/coreplus/coreplus-python-client/
+"""
+
+import random
+
+from combined_table_common import CombinedTable
+from pydeephaven import Session, Table
+
+
+def _db_table(session: Session, namespace: str, table_name: str, is_live: bool) -> Table:
+    """ Get a table from the database on the server.
+
+    Args:
+        session: The session to use.
+        namespace: The namespace of the table.
+        table_name: The name of the table.
+        is_live: True for live table, False for historical table.
+    """
+    tid = random.randint(0, 1000000)
+    session.run_script(
+        f"""_temp_{tid} = db.{"live_table" if is_live else "historical_table"}("{namespace}", "{table_name}")"""
+    )
+    t = session.open_table(f"_temp_{id}")
+    session.run_script(f"""del _temp_{id}""")
+    return t
```

Comment:
Thread safe? Or is it not a concern?

Reply:
I think it isn't a concern at this point.